packages/react-select/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,85 @@
+/// <reference types="@fluentui/react-icons" />
+import * as React from 'react';
+import { makeStyles } from '@fluentui/react-make-styles';
+import { Select } from './Select';
+import { getNativeElementProps, useId } from '@fluentui/react-utilities';
+import { SelectProps } from './Select.types';
+import { ArgTypes } from '@storybook/react';
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '1em',
+    padding: '1em',
+  },
+});
+
+export const SelectExamples = (
```

Comment:
Let's follow the same stories pattern we have in other components. This is primarily so that we can export all the examples to codesandbox cleanly

* one story per file
* each story only imports what can run on codesandbox
* each story is reexported by a main module for storybook